chef/cookbooks/nfs-client/recipes/default.rb

```diff
@@ -100,13 +100,13 @@
     # remove old mount that are not valid anymore
     remove_mount = false
 
-    if not nfs_mounts.has_key?(device)
+    if not nfs_mounts.key?(device)
```

Comment:
Style/Not: Use `!` instead of `not`. (https://github.com/bbatsov/ruby-style-guide#bang-not-not)